pkg/odo/cli/component/update.go

```diff
@@ -112,7 +136,14 @@ func (uo *UpdateOptions) Run() (err error) {
 		if err := component.Update(uo.Context.Client, *uo.cmpConfig, uo.cmpConfig.GetSourceLocation(), stdout); err != nil {
 			return err
 		}
-		log.Successf("The component %s was updated successfully", uo.componentName)
+		if uo.now {
```

Comment:
I think this can be made common to all the component types as effect of `now` is same on all of them? i.e we can move the `if` check out of the `if cmpScrType ==...` in the end of this function call.